CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ option(BUILD_PYTHON "Build the python module" ON)
 
 set(CMAKE_BUILD_TYPE RELEASE)
 set(CMAKE_CXX_STANDARD 17)
+#SET(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -Wall -Werror")
```

Comment:
Was this meant to be left as a comment?  If so, do we still want to leave it in?

Reply:
I meant to remove it, thanks